cloud4rpi/utils.py

```diff
@@ -73,6 +73,13 @@ def to_string(value):
         return str(value)
 
 
+def to_location(value):
+    if isinstance(value, dict):
+        return {x: value[x] for x in ('lat', 'lng')}
```

Comment:
It's not clear why make a dict from the same dict. What's wrong with another keys if we don't use them? Maybe so?
```
if not (isinstance(value, dict) and 'lat' in value and 'lng' in value):
    raise Exception()
return value
```

Reply:
We don't use payload fields except 'lat' and 'lng' on a server side. It's not need to transfer anything else.